test/fork/run-mac-only.test.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const assert = require('assert');
+const utils = require('../utils');
+const appJS = utils.appjs;
+const run = utils.run;
+
+const filenames = [
+  [__dirname + 'some\\\"file', '#!/usr/bin/env node\nconsole.log("OK");'],
+  [__dirname + 'some\ \\file', '#!/bin/sh\necho "OK"'],
+];
+
+if (!process.env.TRAVIS && process.platform !== 'win32') {
```

Comment:
Why not run this on Travis?

Reply:
I don't remember, but it was carried across from the original tests. I suspect it made things blow up, and it was a very, very specific test.